src/pythermalcomfort/jos3_functions/thermoregulation.py

```diff
@@ -15,33 +15,10 @@
 
 from pythermalcomfort.jos3_functions.matrix import NUM_NODES, IDICT, BODY_NAMES
 from pythermalcomfort.jos3_functions import construction as cons
-
-
-_BSAst = np.array(
-    [
-        0.110,
-        0.029,
-        0.175,
-        0.161,
-        0.221,
-        0.096,
-        0.063,
-        0.050,
-        0.096,
-        0.063,
-        0.050,
-        0.209,
-        0.112,
-        0.056,
-        0.209,
-        0.112,
-        0.056,
-    ]
-)
-
+from pythermalcomfort.jos3_functions.construction import Default
 
 def conv_coef(
-    posture="standing",
+    posture=Default.posture,
     v=0.1,
     tdb=28.8,
     t_skin=34.0,
```

Comment:
Should we add t_skin = 34 as a default parameter as well? The same can be done for the above temperature and wind speed event though V and T are less of a Default parameter.

Reply:
I added some other parameters to the default data class, such as t_skin, ta, tr, rh, etc and input those data to the functions instead of inputting the numerical values for us to easily change the code.